workflow/rules/process/network.smk

```diff
@@ -0,0 +1,52 @@
+"""Process gridfinder elements for each box
+
+"""
+
```

Comment:
The snakemake structure for all these process rules is very similar -- it's likely we can have a `process` wildcard that triggers a different `script` depending on its value, allowing us to collapse all these `./workflow/rules/process` rules together and perhaps define which ones we want to run in the config or in the `rule all` equivalent.

Reply:
Interesting. May require quite some restructuring but is a possibility